cli/src/validator_info.rs

```diff
@@ -339,7 +339,7 @@ pub fn process_set_validator_info(
             let instructions = vec![config_instruction::store(
                 &info_pubkey,
                 false,
-                keys,
+                keys.clone(),
```

Comment:
nit: pretty sure these clones are unnecessary

Reply:
Can clone it in the closure, so we save one clone call. Implemented it like this.